src/routes/requestRoute.js

```diff
@@ -0,0 +1,129 @@
+const controller = require('../controllers/requestController');
+const { Router } = require('express');
+const protection = require('../middlewares/check-auth');
+
+const router = Router();
+
+const createR = controller.createRequest;
+const updateR = controller.updateRequest; 
+const deleteR = controller.deleteRequest;
+const getR = controller.getAllRequests;
+
+
+/**
+ * @swagger
+ * /Request:
+ *    post:
+ *      tags: [REQUESTING ACCOMMODATION]
+ *      summary: Authenticated user can request an accommodation in a hotel
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/requests'
+ *      responses:
+ *        "200":
+ *          description: A request schema
+ *
+ * components:
+ *    schemas:
+ *      requests:
+ *        type: object
+ *        required:
+ *          - dateStart
+ *          - dateEnd
+ *          - idRoom
+ * 
+ *        properties:
+ *          dateStart:
+ *            type: string
+ *          dateEnd:
+ *            type: string
+ *          idRoom:
+ *             type: number
+ */
+router.post('/', protection, createR);
+
```

Comment:
Consider validation your request otherwise you can send multiple requests with the same data and also you can even send request with empty data
<img width="1404" alt="Screen Shot 2021-01-25 at 15 13 13" src="https://user-images.githubusercontent.com/54619678/105710677-1cb2bc80-5f20-11eb-9f79-ed24f752ee57.png">
.